.github/workflows/caching-build.yml

```diff
@@ -185,6 +210,8 @@ jobs:
       matrix:
         scheduler: [ scx_bpfland, scx_lavd, scx_layered, scx_rlfifo, scx_rustland, scx_rusty ]
     steps:
+      # prevent cache permission errors
+      - run: sudo chown root /usr/bin/tar && sudo chmod u+s /usr/bin/tar      
       - uses: actions/checkout@v4
       - uses: ./.github/actions/install-deps-action
       # cache virtiofsd (goes away w/ 24.04)
```

Comment:
We can probably add the per scheduler veristat meson script as well to this. That way if there is a verifier error it will be in the test for the scheduler as well as in the integration test.

Reply:
I'd add that separately because that might have flakiness of it's own/I haven't seen that run on CI a lot (i.e. to goal of this is mainly to make CI output more useful while one scheduler is broken + ensure caching works right so queuing doesn't become an issue).